common/utils/src/main/resources/error/error-classes.json

```diff
@@ -804,12 +804,12 @@
     "subClass" : {
       "BOTH_POSITIONAL_AND_NAMED" : {
         "message" : [
-          "A positional argument and named argument both referred to the same parameter."
+          "A positional argument and named argument both referred to the same parameter. Please remove the named argument referring to this parameter."
```

Comment:
These error message changes are specific to SQL named arguments, can you please update the PR title and description to indicate that?